app/util/getICOTokens.js

```diff
@@ -1,22 +1,24 @@
-export const getICOTokens = () =>
-  // Import List. For now return hardcoded
-  [
-    {
-      token: 'RPX',
-      scriptHash: 'AEdAeaa7421adkdaslkea2321qd',
-      supply: 11333311145,
-      name: 'Red Pulse'
-    },
-    {
-      token: 'ONT',
-      scriptHash: 'AEdAeaa7421adkdaslkea2321qd',
-      supply: 11333311145,
-      name: 'Ontology'
-    },
-    {
-      token: 'RNG',
-      scriptHash: 'AEdAeaa7421adkdaslkea2321qd',
-      supply: 11333311145,
-      name: 'Random Test Token'
-    }
-  ]
+import { get } from 'lodash'
+import store from '../store/configureStore'
+
+import ICOTokenList from '../../ICOTokens.json'
+
+export const getICOTokens = () => {
+  const state = store.getState()
+  /* eslint-disable-next-line */
+  const tokens = get(state, 'spunky.settings.data.tokens')
```

Comment:
Everywhere else where we have functions that reference tokens on the state tree they are passed in as function params rather than accessed in this manner... Can we please update this so that if behaves like the rest of the app?